pkg/scheduler/plugins/predicates/predicates.go

```diff
@@ -401,7 +409,16 @@ func (pp *predicatesPlugin) OnSessionOpen(ssn *framework.Session) {
 		// the processing logic needs to be added to the return value result.
 		if predicate.podTopologySpreadEnable {
 			_, status := podTopologySpreadFilter.PreFilter(context.TODO(), state, task.Pod)
-			if !status.IsSuccess() {
+			if status.IsSkip() {
+				taskKey := api.PodKey(task.Pod)
+				if _, ok := skipPlugins[taskKey]; !ok {
+					plugins := sets.New[string]()
+					skipPlugins[taskKey] = plugins
+				}
+				skipPlugins[taskKey].Insert(podTopologySpreadFilter.Name())
+				klog.V(3).Infof("pod(%s/%s) affinity require information is nil, plugin podTopologySpreadFilter is skipped",
```

Comment:
Change log level to 5 to avoid rredundant logs, and so does other plugin's skip check.

Reply:
done